middleware-public/exclusions.js

```diff
@@ -0,0 +1,43 @@
+var async = require('async');
+
+module.exports = function () {
+
+    if (!arguments.length) {
+        throw new Error('Missing parameters! Please provide at least one middleware function.');
```

Comment:
I think this needs some context. Something like `Exclusions middleware is missing parameters. Please provide at least one middleware function.`